org.mixedrealitytoolkit.uxcomponents/Editor/CreateElementMenus.cs

```diff
@@ -210,6 +214,19 @@ private static void CreateActionButton(MenuCommand menuCommand)
             CreateElementFromPath(ActionButtonPath, menuCommand);
         }
 
+        [MenuItem("GameObject/UI/MRTK/Simple Button")]
+        private static void CreateSimpleButton(MenuCommand menuCommand)
+        {
+            GameObject simpleButton = CreateElementFromPath(SimpleButtonPath, menuCommand);
+            GameObject simpleButtonContentGameObject = simpleButton.transform.GetChild(3).gameObject;
```

Comment:
Yes, I do it (in latest commit) with:
```
            foreach (Transform child in simpleButton.transform)
            {
                if (child.name.Equals("Content"))
                {
                    simpleButtonContentGameObject = child.gameObject;
                }
            }
```
**Note 1**:  The reason I didn't use `gameObject.GetComponentsInChildren<TextMeshProUGUI>(true).Where(t => t.name == "Text").First();` initially because SimpleButton prefab doesn't have a TextMeshProUGUI by default under "Content" GameObject so I couldn't query for it as a component.
**Note 2**: The reason why SimpleButton's Content doesn't have children is Mesh's MasterButtonRoot (the source of inspiration for this SimpleButton) doesn't have children under its Content GameObject either.